integration/single/test_basic_function.py

```diff
@@ -15,6 +17,7 @@ class TestBasicFunction(BaseTest):
             "basic_function_openapi",
         ]
     )
+    @skipIf(should_exclude_test_in_region("LambdaEnvVars"), "LambdaEnvVars is not supported in this testing region")
```

Comment:
I could not find the "LambdaEnvVars" in the exclusion file. Does it mean that it should be supported by all regions?
Also, does it mean that all the test cases, should define the functionality name that it got tested by it regardless it will be added to the exclusion file or not?

Reply:
I will remove the "LambdaEnvVars" since it we don't use it in the exclusion file. If we don't want to skip a test in certain region, we don't need to define a service/functionality name in the test case within skipIf decorator.